docs/content/docs/dev/table/materialized-table/statements.md

```diff
@@ -268,10 +287,24 @@ CREATE MATERIALIZED TABLE my_materialized_table_full
     GROUP BY
         p.ds, p.product_id, p.product_name
 ```
+And same materialized table with explicitly specified columns 
+
+```sql
+CREATE MATERIALIZED TABLE my_materialized_table_full (
+    ds, product_id, product_name, avg_sale_price, total_quantity)
```

Comment:
include a data type here? I think specifying both the name and data type is a very common task.